plugins/Editing.jsx

```diff
@@ -272,7 +277,8 @@ class Editing extends React.Component {
         }
         let deleteBar = null;
         if (this.props.editing.action === 'Pick' && this.props.editing.feature && !this.props.editing.changed) {
-            if (!this.state.deleteClicked) {
+            if (!this.state.deleteClicked && (typeof(curConfig.permissions) === 'undefined' || curConfig.permissions.deletable === true)) {
+                // Delete button will appear by default if no permissions are defined in theme editConfig or when deletable permission is set
```

Comment:
This looks wrong, as I read it it will always display the YES/NO buttons rather than hiding the deleteBar completely. Also, easier:

     (curConfig.permissions || {}).deletable === true

Reply:
Or you could just do editPermissions = curConfig.permissions || {} after curConfig = ...;

Reply:
Yes you're totally right. I will hide the deleteBar completely.

I will use `editPermissions`, thanks !
 
I added 
` typeof(curConfig.permissions) === 'undefined' `
in my if statements in order to stay compatible with previous edit config file versions without `permissions`defined. 

If no `permission` is set in edit configuration file, I want that all edit buttons are displayed. 
Do I stay in that way ?

Reply:
The fallback makes sense, but it's shorter to just write 

    const editPermissions = curConfig.permissions || {}

once at the beginning, and then testing 

    editPermissions.delteable/creatable/etc === true

afterwards. No need for typeof.

Reply:
yes I understand :) 
Thanks for your advice